src/styles/index.less

```diff
@@ -165,3 +165,4 @@ Components
 @import "components/tree-list.less";
 @import "components/typography.less";
 @import "components/user-dropdown-menu.less";
+@import "components/collapsible-error-message.less";
```

Comment:
Would be good to alphabetize this import, unless it needs to be applied at the end.